Settings.c

```diff
@@ -838,10 +877,12 @@ Settings* Settings_new(unsigned int initialCpuCount, Hashtable* dynamicMeters, H
 #endif
    this->changed = false;
    this->delay = DEFAULT_DELAY;
-   bool ok = Settings_read(this, this->filename, initialCpuCount);
+   bool checkWritability = true;
```

Comment:
What's the purpose of this variable?
Why not call `Settings_read()` with `true`, `false` and `false`?

Reply:
It's for code readability. Because generally functions with boolean parameters are a bad idea.
But for `Settings_read()` we unfortunately have to add one and it just helps code readability if we use a named boolean variable rather than pass `true`/`false` directly to the functions.

Reply:
`/*param*/true` is always an option …

Reply:
Not a fan of the "`/*param*/ true`" style, but it wouldn't hurt, I guess.